UIKitTeamPlate/UIKitTeamPlate/ClosingViewController.swift

```diff
@@ -4,10 +4,102 @@
 import UIKit
 
 /// Закрывающий экран
```

Comment:
```suggestion
/// Закрывающий экран
```
как понять закрывающий